python/pacemaker/_cts/audits.py

```diff
@@ -279,6 +280,11 @@ def _output_has_core(self, output, node):
 
         return found
 
+    def _find_core_with_coredumpctl(self, node):
+        """Use coredumpctl to find core dumps on the given node."""
+        (_, lsout) = self._cm.rsh(node, "coredumpctl --no-legend")
```

Comment:
I think we should be using `--no-pager`.

Also, `coredumpctl` was added in systemd v195. The `--no-legend` option was added in v196:
https://github.com/systemd/systemd/commit/9a34088

Only one version is affected, and it's 13 years old, so we can ignore it and use `--no-legend` if you want. Or we can introduce some extra parsing or condition.

Reply:
Note that it won't really be a "core file" if found via `coredumpctl`. It will be a line of output like:
```
Sat 2025-01-25 12:22:19 PST 1232012 1000 1000 SIGABRT present      /home/reid/git/pacemaker/tools/.libs/crm_verify                                                                                     1.8M
```

I'm not that worried about the log message though.

Reply:
I think we're good to use `--no-legend`.  I did a quick search through package version lists and it looks like 230-something what's in all the OSes we support.

Reply:
If at any point we want to start requiring a minimum systemd version (when systemd is in use at all), it might allow us to use the SD-Bus interface (in libsystemd) for systemd D-Bus interaction, instead of using the generic D-Bus API. See https://github.com/ClusterLabs/pacemaker/pull/3805#discussion_r1938679436.

Although to be honest, I'm not sure why it exists. It doesn't seem much simpler, if at all.